visualization/app/codeCharta/ui/dialog/dialog.globalSettings.component.ts

```diff
@@ -61,37 +62,48 @@ export class DialogGlobalSettingsController
 	}
 
 	private initDialogOnClick() {
+		this.loadInitGlobal()
+	}
+
+	loadInitGlobal(){
```

Comment:
```suggestion
```

I am not certain if there's a reason for this addition?

Reply:
No, glad that you saw that! I did that when I was testing out a bit, but did not revert that ... 🙃